warehouse/src/table_update/main.py

```diff
@@ -71,6 +75,8 @@ def parse_record(ddb_record: dict) -> Optional[dict]:
         detail_type = "PackageCreated"
         detail["products"] = products
 
+    metrics.add_metric(name=detail_type, unit=MetricUnit.Count, value=1)
```

Comment:
Metrics in other services use camelCase, while this would produce metrics in PascalCase. It's a bit of a nitpick, but for consistency, wouldn't it be better to use camel case here too?

Reply:
My mistake, I have changed the metrics to camelCase